tests/playwright/src/model/pages/play-kube-yaml-page.ts

```diff
@@ -26,16 +28,35 @@ export class PlayKubeYamlPage extends BasePage {
   readonly yamlPathInput: Locator;
   readonly playButton: Locator;
   readonly doneButton: Locator;
+  readonly podmanRuntimeButton: Locator;
+  readonly kubernetesRuntimeButton: Locator;
+  readonly kubernetesContext: Locator;
+  readonly kubernetesNamespaces: Locator;
+  readonly kubernetesButtonCentre: Locator;
+  readonly podmanButtonCentre: Locator;
 
   constructor(page: Page) {
     super(page);
     this.heading = page.getByRole('heading', { name: 'Create pods from a Kubernetes YAML file' });
     this.yamlPathInput = page.getByPlaceholder('Select a .yaml file to play');
+    this.podmanRuntimeButton = page.getByRole('button', { name: 'Podman Container Engine Runtime' });
+    this.podmanButtonCentre = this.podmanRuntimeButton.getByRole('region', { name: 'Podman Button Centre' });
+    this.kubernetesRuntimeButton = page.getByRole('button', { name: 'Kubernetes Cluster Runtime' });
+    this.kubernetesButtonCentre = this.kubernetesRuntimeButton.getByRole('region', {
```

Comment:
Since the button contains nested components and labels, Playwright is unable to click on the button. A workaround is needed to allow the Playwright selector to click on a more specific part of the button.

Reply:
I'm sorry but I understand this, playwright be default navigates to the middle of the button as far as I know, it being nested in the DOM or not has no importance, or at least it shouldn't.